### PR TITLE
Russian language support

### PR DESCRIPTION
**Pull request by Veraprot**

### CODE REVIEW

File: tokens/russian.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+^-$: SUB
+^\*$: MULT
+^/$: DIVIDE
+^;$: SEMICOLON
+^\($: OPEN_PARENTHESYS
+^\)$: CLOSE_PARENTHESYS
+^==$: EQUAL_TO
+^напечатать$: OUTPUT
+^ $: SPACE
+^\н$: NEW_LINE
```
Comment (vgsantoniazzi) [inline tokens/russian.yml]: Can you change this NEW_LINE to `^\n$`? This should match ASCII new line.

File: examples/russian.tl [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  напечатать результат;
+конец
+
+асинхронн число x = 10;
+асинхронн напечатать x;
+
+напечатать "прибавить к x: ";
+
+читатьСтроку к число x;
+напечатать x;
```
Comment (vgsantoniazzi) [inline examples/russian.tl]: Awesome! Tested and is fully working right now! I've updated the `IsWord()` function.

File: tokens/russian.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+################
+# A token might be a variable or function name (AKA an identifier), an operator or a number.
+# USE C++ match_regex SYNTAX TO WRITE NEW TOKENS
+################
```
Comment (vgsantoniazzi) [inline tokens/russian.yml]: Can you translate those comments to Russian as well? The file names would be nice be called like `русский` (.tl and .yml) maybe?

**Review by vgsantoniazzi — COMMENTED** [review]
@Veraprot Can you rebase your branch and test if is working right now on your machine? I've added a commit that supports Russian right now.

File: tokens/russian.yml (hunk 1 above)
Comment (Veraprot) [reply tokens/russian.yml]: just wanted to double check before I make changes. H in Cyrillic is equivalent to N in roman alphabet. should i still make that change?

**Pull request merged**